arelle/plugin/validate/HMRC/ValidateHmrc.py

```diff
@@ -91,13 +100,73 @@
                 re.compile(r".*Paratowyd yn unol â.*darpariaethau.*cwmnïau bach.*"),
             ),
         ),
-    }
+    },
+    LP_ABRID: {
```

Comment:
I see LP.Abrid, but not Co.Abrid? That also should be added even if its not being used directly in this PR.